robotouille/robotouille_env.py

```diff
@@ -74,9 +74,9 @@ def create_robotouille_env(problem_filename, seed=None, noisy_randomization=Fals
         environment_json = _procedurally_generate(environment_json, int(seed), noisy_randomization)
     layout = _parse_renderer_layout(environment_json)
     config_filename = "robotouille_config.json"
+    problem_string, environment_json = builder.build_problem(environment_json) # IDs objects in environment
```

Comment:
I moved this so that the environment_json can be updated to have robot1, robot2, etc, since the original environment json does not know which player is which. With the ids, we are able to draw players by their id